### PR TITLE
fix(fast-form-generator-react): update + glyph positioning

### PR DESCRIPTION
**Pull request by nalogiudice**
Closes [#599](https://github.com/Microsoft/fast-dna/issues/599)

<body>

<footer>
 

For [details on formatting](https://github.com/Microsoft/fast-dna/wiki/pull-request-guidance) pull requests.

### CODE REVIEW

File: packages/fast-form-generator-react/src/utilities/form-input.style.ts [hunk 1]
```diff
@@ -466,7 +466,8 @@ export function applyAddItemStyle(): ICSSRules<{}> {
             background: pinkPlus,
             width: toPx(16),
             height: toPx(16),
-            left: "0"
+            left: "0",
+            top: toPx(11)
```
Comment (chrisdholt) [inline packages/fast-form-generator-react/src/utilities/form-input.style.ts]: fixing to 11px threw this off in chrome for me when testing. If we need to absolutely position this for all browsers, let's consider a more standard approach:

```
top: "50%",
transform: "translateY(-50%)"
```

**Comment by awentzel** [comment]
Scope should include hyphens if a package name per documentation

File: packages/fast-form-generator-react/src/utilities/form-input.style.ts (hunk 1 above)
Comment (nalogiudice) [reply packages/fast-form-generator-react/src/utilities/form-input.style.ts]: Wait really? I saw no change in Chrome positioning at all.

File: packages/fast-form-generator-react/src/utilities/form-input.style.ts [hunk 2]
```diff
@@ -466,7 +467,9 @@ export function applyAddItemStyle(): ICSSRules<{}> {
             background: pinkPlus,
             width: toPx(16),
             height: toPx(16),
-            left: "0"
+            left: "0",
+            top: "50%",
+            transform: "translateY(-50%)"
```
Comment (eljefe223) [inline packages/fast-form-generator-react/src/utilities/form-input.style.ts]: I assume this guarantees that the glyph is positioned center vertically in its parent?

Comment (nalogiudice) [reply packages/fast-form-generator-react/src/utilities/form-input.style.ts]: Correct.

**Pull request merged**